apps/labs/posts/numpy-python-api-cleanup.md

```diff
@@ -0,0 +1,345 @@
+---
+title: 'Python API cleanup for NumPy 2.0'
```

Comment:
Since it's a title, a bit of an attempt to pique the interest of potential readers is always useful. How about something like _"Refining NumPy's Python API for its 2.0 release"_ or _"A journey in API design for NumPy 2.0"_?

Reply:
Sure - I chose the first one.